checkov/bicep/checks/param/azure/SecureStringParameterNoHardcodedValue.py

```diff
@@ -18,8 +18,7 @@ def scan_param_conf(self, conf: ParameterAttributes) -> CheckResult:
         if not any(decorator["type"] == "secure" for decorator in conf["decorators"]):
             # if the decorator '@secure()' is not set, then it is a normal string
             return CheckResult.UNKNOWN
-
-        if conf["default"]:  # should be missing, or an empty string
+        if conf.get("default"):  # should be missing, or an empty string
```

Comment:
Yeah, that's the last check I am trying to figure out 😅 
Since `conf` is of type `ParameterAttributes` which is a `TypedDict` it didn't accept gracefully the addition of fields